### ISSUE TITLE
Setting to disable preview thumbnails

### ISSUE THREAD

**Issue by sicherist** [cboxdoerfer/fsearch#499]
**Is your feature request related to a problem? Please describe.**
Fsearch shows a thumbnail of pdf-files in the search results. I find it rather confusing when I'm looking for a pdf, expecting a pdf-icon but instead I see a lot of slightly different looking white-ish icons that are too small to see details. I would prefer to have the same generic icon per file-type.

**Describe the solution you'd like**
I would love to have a setting in the preferences where preview-thumbnails can be turned off.

**Describe alternatives you've considered**

**Additional context**

**Comment by cboxdoerfer** [comment]
Yes, that's what I thought as well. I'll add an option for that.
